src/thinblock.cpp

```diff
@@ -28,6 +28,22 @@ extern CCriticalSection cs_thinblockstats;
 extern CCriticalSection cs_orphancache;
 extern map<uint256, COrphanTx> mapOrphanTransactions GUARDED_BY(cs_orphancache);
 
+static string formatInfoUnit(double value)
+{
+    static const char *units[] = {"B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"};
+    int i = 0;
+    while (value > 1000.0 && i < 7)
```

Comment:
7 should not be hard-coded.  0...6 is probably enough anyway, no need for the ZB and YB